cvat/apps/engine/serializers.py

```diff
@@ -94,7 +94,8 @@ def __init__(self, *, model=models.Label, url_filter_key, source='get_labels', *
 
 
 class JobsSummarySerializer(_CollectionSummarySerializer):
-    completed = serializers.IntegerField(source='completed_jobs_count', default=0)
+    completed = serializers.IntegerField(source='completed_jobs_count', allow_null=True)
+    validation = serializers.IntegerField(source='validation_jobs_count', allow_null=True)
```

Comment:
Why are these fields `nullable` and `required` but the `count` field has a `default` value and isn't `required`?